flashlight/app/objdet/CMakeLists.txt

```diff
@@ -1,56 +1,24 @@
 cmake_minimum_required(VERSION 3.10)
 
-add_library(
-  flashlight-app-objdet
-  ""
-  )
-
-target_link_libraries(
-  flashlight-app-objdet
-  PUBLIC
-  ${gflags_LIBRARIES}
-  ${GLOG_LIBRARIES}
-  flashlight
-  flashlight-app-common
-  )
-
-target_include_directories(
-  flashlight-app-objdet
-  PUBLIC
-  ${gflags_INCLUDE_DIRS}
-  ${GLOG_INCLUDE_DIRS}
-  )
-
-# Dataset
-include(${CMAKE_CURRENT_LIST_DIR}/dataset/CMakeLists.txt)
-
-# Models
-include(${CMAKE_CURRENT_LIST_DIR}/models/CMakeLists.txt)
-
-# Criterion
-include(${CMAKE_CURRENT_LIST_DIR}/criterion/CMakeLists.txt)
-
-# NN
-include(${CMAKE_CURRENT_LIST_DIR}/nn/CMakeLists.txt)
-
 add_executable(
   fl_img_imagenet_resnet50_backbone
   ${CMAKE_CURRENT_LIST_DIR}/examples/ImageNetResnet50Backbone.cpp)
 target_link_libraries(
   fl_img_imagenet_resnet50_backbone
-  flashlight-app-imgclass
-  flashlight-app-objdet)
+  fl_pkg_vision
+  fl_pkg_runtime
+  )
 set_executable_output_directory(fl_img_imagenet_resnet50_backbone ${FL_BUILD_BINARY_OUTPUT_DIR}/objdet)
 
 add_executable(fl_img_coco_detr ${CMAKE_CURRENT_LIST_DIR}/examples/CocoDetr.cpp)
-target_link_libraries(fl_img_coco_detr flashlight-app-objdet)
+target_link_libraries(
+  fl_img_coco_detr 
+  fl_pkg_vision
```

Comment:
This is so much better already. Not required, but if we wanted to, we could also create a function like `fl_register_objdet_binary` which links the libraries, sets the executable output directory, etc.

Reply:
That's a good idea. I'll keep it as is for now, since it would only be two calls, but I'll keep this in my mind in the future, especially if we add more.